docs/topics/downloader-middleware.rst

```diff
@@ -190,6 +190,12 @@ The following settings can be used to configure the cookie middleware:
 * :setting:`COOKIES_ENABLED`
 * :setting:`COOKIES_DEBUG`
 
+Notice that if the :class:`~scrapy.http.Request` 
+has ``meta['dont_merge_cookies']`` present then 
+despite the value of :setting:`COOKIES_ENABLED` the cookies received in the
```

Comment:
This has a small readability flow issue: we haven't explained what COOKIES_ENABLED does yet. 
What do you think about moving it somewhere after COOKIES_ENABLED docs?
Alternatively, we can just mention that middleware is controlled by this key, and refer to Request docs.